### ISSUE TITLE
Unable to resize window (1.4.2)

### ISSUE THREAD

**Issue by Jamezhall** [Kilian/fromscratch#74]
Gday,
Couple of things

- Main issue: It seems, with the latest update, the ability to resize the window has been removed. Is this intentional? See below:
![rtvideo - 2018-09-27 09 26](https://user-images.githubusercontent.com/19564725/46114908-1d47c500-c238-11e8-9626-1b409e1a8a67.gif)

- There is a white border on the dark theme when the window is active. I dont remember it being there. Should this be a different color? or is this just my memory being broken. (In case it helps, my windows default is blue for an active window, so this must be an electron change I assume?)
![image](https://user-images.githubusercontent.com/19564725/46114937-41a3a180-c238-11e8-8e00-775464511a88.png)

- https://fromscratch.rocks/latest.json has not been updated and did not trigger an in app alert :)

OS: win10 pro
ver: 1.4.2

Let me know if you need any more info.
Thanks for keeping up the development of this.. I love this little app! and use it daily for everything!

**Comment by Kilian** [comment]
Can you check if this is also the case with 1.4.1 or 1.4.0? They're still available from here: https://github.com/Kilian/fromscratch/releases

I always wait a couple of weeks for the in-app alert to shake out any remaining bugs.

**Comment by beppe9000** [comment]
Same issue! I am on windows server 2016.  
And the maximize button is also disabled.
The window itself lacks that "native" feel.

**Comment by Kilian** [comment]
@beppe9000 Can you let me know if this is still the case in 1.4.3?

**Comment by PSZombie** [comment]
I am having the same issue with version 1.4.3.

**Comment by Kilian** [comment]
this will be fixed in the next release!

**Comment by Jamezhall** [comment]
Looking forward to it :)

**Comment by andreapx** [comment]
Right now the size can be changed in the file "C:\Users\%USERNAME%\.fromscratch\windowstate" (for Windows, for other platforms I don't know where the file "windowstate" is located).
`{"isMaximized":false,"bounds":{"x":196,"y":225,"width":**1000**,"height":**550**}}`

**Comment by Jamezhall** [comment]
Thanks @andreapx!

Also I just noticed I can't snap the app window to the screen (win + left, win + right etc)
And all snap settings are enabled. (https://imgur.com/g6np0A3)
I'm guessing this is also something to do with the fact the window cannot be resized currently (v1.4.3).

**Comment by Jamezhall** [comment]
@Kilian Heyo, Just wanted to check if this window resize fix was scheduled for release soonish? Still using the app daily! Thanks for making a great little app!

**Comment by tcjdesign** [comment]
I just updated from 1.3.0 to 1.4.3 and encountered this issue. It's a deal breaker so I had to downgrade back to 1.3.0.

The ugly white border is bad, but losing the ability to maximize without going fullscreen and covering the task bar is antithetical to how I use fromscratch, which is super good and has changed how I keep track of things in the theater.

**Comment by Jamezhall** [comment]
Yeah I still use this app daily. Would really love an update! But I do understand it is only a side project.. I might downgrade as well...

**Comment by taylorcjensen** [comment]
Just wanted to follow up and say this issue still exists in latest as of now (1.4.3) and I am using v1.4.0 to avoid it.
